src/components/molecules/LabelsSelect/LabelsSelect.tsx

```diff
@@ -17,7 +17,7 @@ import {composeLabels} from './utils';
 
 type LabelsSelectProps = {
   onChange?: (value: readonly Option[]) => void;
-  defaultLabels?: readonly Option[];
+  defaultLabels?: Record<string, string> | Option[];
```

Comment:
It would be better to keep a single format. Do we actually want `Option[]`? Looking at `composeLabels`, it is actually expecting `Record<any, Option>`.